docs/site/Discovering-models-from-relational-databases.md

```diff
@@ -0,0 +1,49 @@
+---
+lang: en
+title: 'Discovering models from relational databases'
+keywords: LoopBack 4.0, LoopBack-Next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Discovering-models-from-relational-databases.html
```

Comment:
IIRC, Juggler supports several discovery modes. Besides parsing SQL schema, it can also define models based on example data. I think it would be great to implement support for this feature at some point in the future (but not in this pull request).

To keep our doors open, I am proposing to rename this page and leave out the part about relational databases.
- file name: `Discovering models.md`
- permalink: `/doc/en/lb4/Discovering-models.md`

Or perhaps also rename the page to "Model discovery" (`Model-discovery.md)? IDK.

Reply:
done